app/login.tsx

```diff
@@ -0,0 +1,348 @@
+
+export default function LoginScreen() {
+  const [showPassword, setShowPassword] = useState(false)
+  const [isLoading, setIsLoading] = useState(false)
+
+  // Cargar fuentes Poppins
+  const [fontsLoaded] = useFonts({
+    Poppins_400Regular,
+    Poppins_500Medium,
+    Poppins_600SemiBold,
+    Poppins_700Bold,
+  })
+
+  const {
+    control,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<LoginFormData>({
+    resolver: zodResolver(loginSchema),
+  })
+
+  if (!fontsLoaded) {
+    return null // O un componente de carga
+  }
+
+  const onSubmit = async (data: LoginFormData) => {
+    setIsLoading(true)
+    try {
+      // Aquí iría la lógica de autenticación
+      console.log('Login data:', data)
+      Alert.alert('Success', 'Login successful!')
+      // router.replace('/(tabs)')
+    } catch (error) {
+      Alert.alert('Error', 'Login failed. Please try again.')
+    } finally {
+      setIsLoading(false)
+    }
+  }
+
+  const handleSocialLogin = (provider: 'facebook' | 'google' | 'apple') => {
+    Alert.alert('Social Login', `${provider} login would be implemented here`)
+  }
+
+  const handleForgotPassword = () => {
+    Alert.alert(
+      'Forgot Password',
+      'Password reset functionality would be implemented here'
+    )
+  }
+
+  const handleSignUp = () => {
+    Alert.alert('Sign Up', 'Sign up screen would be implemented here')
+  }
+
+  return (
+    <SafeAreaView style={styles.container}>
+      {/* Title */}
+      <View style={styles.titleContainer}>
+        <Text style={styles.title}>Sign In</Text>
+      </View>
+
+      {/* Form */}
+      <View style={styles.formContainer}>
+        <View style={styles.formContent}>
+          {/* Email Input */}
+          <View style={styles.inputContainer}>
+            <Controller
+              control={control}
+              name='email'
+              render={({ field: { onChange, onBlur, value } }) => (
+                <TextInput
+                  style={styles.input}
+                  placeholder='E-mail'
+                  placeholderTextColor='#8E8E93'
+                  value={value}
+                  onChangeText={onChange}
+                  onBlur={onBlur}
+                  keyboardType='email-address'
+                  autoCapitalize='none'
+                  autoCorrect={false}
+                />
+              )}
+            />
+            {errors.email && (
+              <Text style={styles.errorText}>{errors.email.message}</Text>
+            )}
+          </View>
+
+          {/* Password Input */}
+          <View style={styles.inputContainer}>
+            <View style={styles.passwordContainer}>
+              <Controller
+                control={control}
+                name='password'
+                render={({ field: { onChange, onBlur, value } }) => (
+                  <TextInput
+                    style={styles.input}
+                    placeholder='Password'
+                    placeholderTextColor='#8E8E93'
```

Comment:
Please check if you can style the placeholder in the styles.input? If not, add a constan in colors file so you can use it anywhere.